Update fastcheck and linters configurations (#376)
renuo/applications-setup-guide: ruby_on_rails/linting_and_automatic_check.md

## Brakeman

> Brakeman comes byb default with Rails. Add it to the `bin/fastcheck` script.

Reviewer: ```suggestion
Brakeman comes by default with Rails. Add it to the `bin/fastcheck` script.
```

```
bundle exec brakeman -q -z --no-summary --no-pager
```

## Mdl

> An optional check for markdown files. You can include it or not. Discuss within your team.

Reviewer: Maybe add example usage of mdl?
```
bundle exec mdl $(find . -name '*.md'|grep -E -v '(SUMMARY.md|vendor/)')
```

Author: The template includes an example. I don't want to duplicate it here

## SCSS lint

> To lint the SASS/SCSS files in our project we are going to use the `stylelint` npm package.

Reviewer: ```suggestion
To lint the SASS/SCSS files in our project you can use the `stylelint` npm package.
```
